manifests/rhoai/shared/odhdashboardconfig/odhdashboardconfig.yaml

```diff
@@ -1,41 +1,14 @@
-# ODHDashboardConfig to enable the notebook-controller feature and add notebook sizes
+# Read the README before modifying
 apiVersion: opendatahub.io/v1alpha
 kind: OdhDashboardConfig
 metadata:
-  creationTimestamp: null
   name: odh-dashboard-config
+  annotations:
+    opendatahub.io/managed: "false"
 spec:
   dashboardConfig:
-    disableBYONImageStream: false
-    disableClusterManager: false
-    disableISVBadges: false
-    disableInfo: false
-    disableSupport: false
+    # Overrides only -- see blankDashboardCR
```

Comment:
@christianvogt okay, I'm pulling the ripcord -- we are finally moving to just defaults in code & overrides in the file.